packages/mdc-button/_mixins.scss

```diff
@@ -68,8 +70,12 @@
   }
 }
 
-@mixin mdc-button-corner-radius($corner-radius) {
-  border-radius: $corner-radius;
+@mixin mdc-button-shape-radius($radius, $rtl-reflexive: false) {
+  @include mdc-shape-radius(mdc-shape-resolve-percentage-radius($mdc-button-height, $radius), $rtl-reflexive);
+
+  &.mdc-button--dense {
```

Comment:
This makes it impossible to customize the corner radius of dense and non-dense buttons independently, which was previously possible with the old mixin (you'd just apply it to whichever one you wanted). I could see where people might want to do that if they're using an absolute value... though I wonder if it's ever expected to mix and match dense and non-dense variants?

The same is true of e.g. FAB in #3552... and I'd expect it to be true of text field as well, though I don't see it there, which makes me suspect we're not treating this consistently.

I think we need to think through how we want to support dense vs. non-dense variants WRT shape. I realize that for components with fixed heights it needs to be treated separately for purposes of supporting percentage, but I'm not sure we should be forcing different-sized variants to all use the same radius values.

Reply:
As discussed, we apply same shapes for all component variants. User can override a specific variant as a one-off.